docs/spectrum/run_spectrum_analysis.py

```diff
@@ -0,0 +1,28 @@
+from astropy.coordinates import SkyCoord, Angle
```

Comment:
Add one-line comment what this is at the top of the script.
It's always useful to have that one line ... people copy files around or just share a link to the script and not the docs.